src/ed25519/mod.rs

```diff
@@ -1,7 +1,9 @@
 use crate::*;
-use std::convert::TryFrom;
-
-#[derive(Debug, PartialEq, Clone)]
+use std::{
+    convert::TryFrom,
+    hash::{Hash, Hasher},
+};
+#[derive(Debug, Clone)]
```

Comment:
Another overly eager whitespace removal?

Reply:
It would appear so. Thanks